cmd/manager

```diff
@@ -1 +0,0 @@
-/home/nferraro/gopaths/camel-k/src/github.com/apache/camel-k/cmd/kamel
```

Comment:
Probably not in the scope of that PR.

Reply:
right, should I re-work the PR or leave as it is ? I think this file need to be removed in any case

Reply:
Don't bother, it's been removed with #986.